demo/README.md

```diff
@@ -0,0 +1,60 @@
+
+### Goal: Bring up Backend client for Fabric Auction demo network
+
+Some dependencies are currently hardcoded. This is work in progress.  As the scripts to stand up the demo application evolve, these will be modified. Example: NETWORK_NAME used in following instructions.
+
+Usage:
+```
+## Build image once
+cd fabric-private-chaincode/demo/client/backend
+docker build -t auction_backend  .
+```
+
+##  Register auction application users (bidders and auctioneers) with Certificate Authority
+```
+cd fabric-private-chaincode/demo/client/backend
+./registerUsers.sh
+```
+
+## Run client
+```
+export NETWORK_NAME=fabric-fpc_basic
```

Comment:
after my PR this will be fabricfpc-fpc_basic as docker-compose in some cases squashes anything which is not alphanumeric

Reply:
Added note to README and a reference to docker-compose/README - to explain the dependency on the network setup - FPC or non-FPC.